### PR TITLE
Update the description & comment

### PR DESCRIPTION
**Pull request by WangHaizhen**
1. Optimize description in batch-norm.md & conv-layer.md
2. Update comment in linear-regression-scratch.md

### CODE REVIEW

**Comment by astonzhang** [comment]
Thanks. For consistency, we do have period at the end of a code comment. Could you resolve it?

**Comment by WangHaizhen** [comment]
Okay, I didn't notice that before, but now I've restored the period at the end of the code comment

File: chapter_convolutional-neural-networks/batch-norm.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 ## 批量归一化层
 
-对全连接层和卷积层做批量归一化的方法稍有不同。下面我们将分别介绍这两种情况下的批量归一化。
+对全连接层和卷积层做批量归一化的方法稍有不同，下面我们将分别介绍这两种情况下的批量归一化。
```
Comment (astonzhang) [inline chapter_convolutional-neural-networks/batch-norm.md]: We tried to keep "one sentence, one meaning". For technical discussions, short sentences are easier for readers.

File: chapter_convolutional-neural-networks/batch-norm.md [hunk 2]
```diff
@@ -156,7 +156,7 @@ net[1].gamma.data().reshape((-1,)), net[1].beta.data().reshape((-1,))
 
 ## Gluon实现
 
-相比于我们刚刚自己定义的`BatchNorm`类，Gluon中`nn`模块定义的`BatchNorm`类使用起来更加简单。它不需要指定自己定义的`BatchNorm`类中所需的`num_features`和`num_dims`参数值。在Gluon中，这些参数值都将通过延后初始化来自动获取。下面我们用Gluon实现使用批量归一化的LeNet。
+相比于我们刚刚自己定义的`BatchNorm`类，Gluon中`nn`模块定义的`BatchNorm`类使用起来更加简单，它不需要指定自己定义的`BatchNorm`类中所需的`num_features`和`num_dims`参数值。在Gluon中，这些参数值都将通过延后初始化来自动获取，下面我们用Gluon实现使用批量归一化的LeNet。
```
Comment (astonzhang) [inline chapter_convolutional-neural-networks/batch-norm.md]: We tried to keep "one sentence, one meaning". For technical discussions, short sentences are easier for readers.

File: chapter_convolutional-neural-networks/conv-layer.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # 二维卷积层
 
-卷积神经网络是含有卷积层（convolutional layer）的神经网络。本章中卷积神经网络均使用最常见的二维卷积层。它有高和宽两个空间维度，常用来处理图像数据。本节中，我们将介绍简单形式的二维卷积层的工作原理。
+卷积神经网络是含有卷积层（convolutional layer）的神经网络。本章中卷积神经网络均使用最常见的二维卷积层，它有高和宽两个空间维度，常用来处理图像数据。本节中，我们将介绍简单形式的二维卷积层的工作原理。
```
Comment (astonzhang) [inline chapter_convolutional-neural-networks/conv-layer.md]: We tried to keep "one sentence, one meaning". For technical discussions, short sentences are easier for readers.

File: chapter_deep-learning-basics/linear-regression-scratch.md [hunk 4]
```diff
@@ -64,10 +64,12 @@ plt.scatter(features[:, 1].asnumpy(), labels.asnumpy(), 1);
 def data_iter(batch_size, features, labels):
```
Comment (astonzhang) [inline chapter_deep-learning-basics/linear-regression-scratch.md]: We use inline comments to keep each code block small: the book will be thinner :)
Can you revert changes in this md file?

Comment (WangHaizhen) [reply chapter_deep-learning-basics/linear-regression-scratch.md]: Oh... I see. I've restored the relevant changes.

**Comment by astonzhang** [comment]
非常感谢你对《动手学深度学习》一书的贡献。等明年初纸质书出版时，出版社将为贡献者制作并赠送本书的专享版。我们也希望在本书致谢（http://zh.diveintodeeplearning.org/chapter_introduction/preface.html ）中列出你的姓名（出版社希望这周五12月14号前将这部分内容定稿。）能将你的姓名、Github ID 和邮寄地址发送至 gluonbook [AT] gmail [dot] com吗？谢谢。

**Comment by WangHaizhen** [comment]
已经回复邮件，请查收！

**Pull request merged**